backend.py

```diff
@@ -504,15 +504,15 @@ def record_log(self):
         self.log.append(self.robots_as_dict())
 
     @classmethod
-    def get_start_state(cls, map_name):
+    def get_start_state(cls, map_name, players):
```

Comment:
Šlo by z toho udělat nepovinný argument, aby původní kód (jako testy) fungoval dál.

Reply:
```suggestion
    def get_start_state(cls, map_name, players=None):
```

Reply:
Mělo by jít to všude změnit na nepovinné argumenty (`=None`), s tím že `None` pak znamená že se použije maximální počet hráčů.